spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -140,12 +139,30 @@
       default_params
     end
 
-    include_context 'setup cmdb'
+    include_context 'trusted external data test setup'
 
     it 'uses the new oauth token', skip: skip_oauth_tests do
       result = trigger_puppet_run(master)
       trusted_json = parse_json(result.stdout, 'trusted_json')
       expect(trusted_json['external']['servicenow']['fqdn']).to eql(master.uri)
     end
   end
+
+  context 'dry testing the servicenow.rb script fails in the trusted external data setup' do
```

Comment:
Ideally this should be a unit test but since we're using `Exec` to do this, I think it's best left as an acceptance test.